roles/admin/files/haproxy.ctmpl

```diff
@@ -0,0 +1,58 @@
+{{- define "SERVICES_FQDN" -}}
+    services.mozilla.community
```

Comment:
Let's keep our public jenkins (and other services in the future) under services.mozilla.community - we should use ops.mozilla.community instead.